tests/golden/vault/common.bats

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "vault bad path" {
+  run coffer view "$(echo -e "bad\npath")"
+
+  assert_failure
+  assert_output --partial - <<EOF
+Internal backend error:
```

Comment:
> Internal backend error:

This isn't an "internal" error either.

Internal errors are (most likely) bugs in our code or in any internal component, like vault. But, in this case, the error was due to bad data entered by the user, so it's not an internal error.

Reply:
done